package.json

```diff
@@ -26,7 +26,8 @@
     "cordova:prepare": "cordova prepare",
     "cordova:build": "cordova build",
     "cordova:build:iso": "cordova build ios",
-    "cordova:build:android": "cordova build android"
+    "cordova:build:android": "cordova build android",
+    "mm": "./node_modules/.bin/mm"
```

Comment:
seems like the problem arised because of naming module name is mongodb-migrations not mm.